Library/Formula/rtf2html.rb

```diff
@@ -0,0 +1,17 @@
+class Rtf2html < Formula
+  homepage "http://sourceforge.net/projects/rtf2html-lite/"
+  url "https://downloads.sourceforge.net/project/rtf2html-lite/RTF%202%20HTML%20Lite%20-%20Sources/2008-10-03%2C%20first%20LGPL%20release/rtf2html-lite-20081003.tar.bz2"
+  sha1 "eec3b29633f44290fcab3acd5336462cfc737140"
+  version "20081003"
+
+  def install
+    system "make"
+    bin.install "rtf2html"
```

Comment:
There is, but the Makefile is hard-coded to copy the binary straight to /usr/local/bin